apps/src/lib/levelbuilder/course-editor/CourseEditor.js

```diff
@@ -263,17 +274,30 @@ export default class CourseEditor extends Component {
         </CollapsibleEditorSection>
 
         <CollapsibleEditorSection title="Teacher Resources">
+          {this.props.migratedTeacherResources && (
+            <input
+              type="hidden"
+              name="resourceIds[]"
+              value={this.props.migratedTeacherResources.map(r => r.id)}
+            />
+          )}
           <div>
-            Select the Teacher Resources buttons you'd like to have show up on
-            the top of the course overview page
+            <div>
+              Select the Teacher Resources buttons you'd like to have show up on
```

Comment:
I know this isn't new but this text looks like its out of date. Looks like when we updated the course teacher resource to be a dropdown like script we didn't update this text to match.

Reply:
It's the same text as is on the script edit page? I'm happy to change it but I'm not sure what the best wording is?

Reply:
I think "Select the Teacher Resources you'd like to have show up in the dropdown at the top of the course overview page"